src/components/results/_pagination.scss

```diff
@@ -38,81 +38,81 @@
   @include breakpoint($bp2) {
     margin-right: rhythm(.25);
   }
+}
 
-  // Link styles
-  a {
-    @include trans();
-    @include adjust-font-size-to(.9em, 1);
-    font-weight: 700;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    border: 0;
-    text-align: center;
-    padding: rhythm(.25);
-
-    @include breakpoint($bp0) {
-      background-color: $c-secondary;
-      color: $white;
-      border-radius: 3px;
-      width: rhythm(1.5);
-      height: rhythm(1.5);
-      padding: 0;
-    }
+// Active page
+.search-pager__item.is-active .search-pager__item-link {
+  @include adjust-font-size-to($text, 1);
 
-    @include breakpoint($bp2) {
-      @include adjust-font-size-to($label, .9);
-      width: rhythm(1.2);
-      height: rhythm(1.2);
-    }
+  @include breakpoint($bp0) {
+    background-color: $link-color;
+    color: $white;
 
     &:hover,
     &:active {
-      text-decoration: underline;
-      color: $white;
-      background-color: $c-secondary;
-
-      @include breakpoint($bp0) {
-        @include adjust-font-size-to($text, 1);
-        background-color: $c-secondary-dark;
-      }
-
-      svg {
-        fill: $c-primary-dark;
-      }
+      background-color: $c-primary-dark;
     }
+  }
+}
 
-    svg {
-      height: 1.3em;
-      fill: $white;
-      display: block;
+.search-pager__item.not-active .search-pager__item-link {
+  display: none;
 
-      @include breakpoint($bp0) {
-        height: 1em;
-      }
-    }
+  @include breakpoint($bp0) {
+    display: inline-block;
+  }
+}
+
+// Link styles
+.search-pager__item-link {
+  @include trans();
+  @include adjust-font-size-to(.9em, 1);
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 0;
+  text-align: center;
+  padding: rhythm(.25);
+
+  @include breakpoint($bp0) {
+    background-color: $c-secondary;
+    color: $white;
+    border-radius: 3px;
+    width: rhythm(1.5);
+    height: rhythm(1.5);
+    padding: 0;
+  }
+
+  @include breakpoint($bp2) {
+    @include adjust-font-size-to($label, .9);
+    width: rhythm(1.2);
+    height: rhythm(1.2);
   }
 
-  // Active page
-  &.is-active a {
-    @include adjust-font-size-to($text, 1);
+  &:hover,
+  &:active {
+    text-decoration: underline;
+    color: $white;
+    background-color: $c-secondary;
 
     @include breakpoint($bp0) {
-      background-color: $link-color;
-      color: $white;
+      @include adjust-font-size-to($text, 1);
+      background-color: $c-secondary-dark;
+    }
 
-      &:hover,
-      &:active {
-        background-color: $c-primary-dark;
-      }
+    svg {
+      fill: $c-primary-dark;
     }
   }
 
-  &.not-active {
-    display: none;
+  svg {
+    height: 1.3em;
+    fill: $white;
+    display: block;
 
     @include breakpoint($bp0) {
-      display: inline-block;
+      height: 1em;
```

Comment:
@nstriedinger can you walk me through this change too?  Does this just happen to look like a change in the css rule because of how things were rearranged or is an actual change?